### PR TITLE
New blog post: status-update-summer-2015

### PR DESCRIPTION
**Pull request by will-moore**
New blog post about progress of OMERO.figure over the summer.

### CODE REVIEW

File: _posts/2015-08-03-staus-update-summer-2015.markdown [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+that doesn't rely on the Backbone.js library on which OMERO.figure is built.
+This is not quite finished, but you can [try the demo here](http://will-moore.github.io/shape-editor-js/).
+
+Once this is ready, I will move back to OMERO.figure development.
+Initially the focus will be a dialog box that shows a 'full image viewer' with the ability
+to draw and edit simple shapes that are then displayed in the figure.
+A crucial part of this functionality will be the ability to add shapes to exported figures, both in
+the PDF and TIFF formats.
+Ideally ROIs will be kept as atomic entities in the PDF figures, allowing them to be
+individual edited in postscript tools such as Adobe Illustrator. However, this may
```
Comment (mtbc) [inline _posts/2015-08-03-staus-update-summer-2015.markdown]: individually

Comment (mtbc) [inline _posts/2015-08-03-staus-update-summer-2015.markdown]: I doubt they actually go via PostScript

**Comment by will-moore** [comment]
Addressed Mark's suggestions and Jason's "Let’s be a bit more definite in the last sentence.  Maybe preliminarily target for 5.2.0-m1?". Hopefully this is looking good now @hflynn?

**Comment by will-moore** [comment]
@sbesson / @jburel OK to merge this?

**Comment by hflynn** [comment]
I'm just going to look at it now, give me a tick

**Comment by jburel** [comment]
@will-moore: since it is now part of the openmicroscopy. I will be good to update the github and twitter links. For twitter we can have both.

**Comment by will-moore** [comment]
@jburel Links updated. @hflynn Maybe needs one more build to test these, thanks!

**Comment by hflynn** [comment]
All looks fine on staging, good to merge from me.

**Comment by will-moore** [comment]
@jburel OK to merge?

**Comment by jstitlow** [comment]
Is it possible to give permission for other users to update files in .Figure?

**Comment by joshmoore** [comment]
Hi @jstitlow -- perhaps something like https://github.com/ome/figure/issues/95 ? Can you comment there if so or if not, open a new issue? https://github.com/ome/figure/issues/new Cheers.

**Comment by jstitlow** [comment]
Thanks for the quick reply, will leave the comment there.

**Comment by jstitlow** [comment]
Another thought.. Is anyone looking in to 3-D rendering in OMERO with a function to export movies? Maybe this is a new project, we can call it OMERO.Movie?!

**Comment by joshmoore** [comment]
@jstitlow, regarding 3-D rendering, I've add a card: https://trello.com/c/QKtSahbg/88-3d-rendering if you'd like to comment further. At the moment, however, you'd have to make use of an external tool like ImageJ or Icy which can connect to OMERO.

**Comment by will-moore** [comment]
@jstitlow You can currently export an image as a movie of 2D planes, and I'm considering the export of a time-lapse figure as a Movie https://github.com/ome/figure/issues/51 but not 3D I'm afraid.

**Comment by jstitlow** [comment]
Thanks, #51 looks cool! Finding a way to export the Figure in .avi as
multiple annotated movies playing simultaneously would be incredible.

For 3D rendering the images we want to display are single time points. We
want to project the images in 3D in Figure and display rotated views as we
would in Imaris.

**Pull request merged**